tool/planet/agent.go

```diff
@@ -349,7 +349,7 @@ func runAgent(config agentConfig) error {
 	}
 	config.agent.Tags["role"] = string(config.monitoring.Role)
 
-	clientset, err := getKubeClientFromPath(constants.KubectlConfigPath)
+	clientset, err := getKubeClientFromPath(constants.KubeletConfigPath)
```

Comment:
Have you been able to verify whether this works as expected on non-master nodes?

Reply:
Yeah, using `KubeletConfigPath` allows permissions to query Kubernetes Nodes. There is also a NodeStatus check that runs on non-master nodes that is able to query Kubernetes Nodes using the same kubeconfig.